recipes/configs/llama3_2/1B_full_single_device.yaml

```diff
@@ -19,11 +19,16 @@
 #
 # This config works only for training on single device.
 
+output_dir: /tmp/llama_3_2_1b/full_single_device
```

Comment:
What's the purpose of these config changes? Aren't these just a subset of the broader config changes in #2107?

Reply:
i changed only these 2 configs here for testing. Then i created the other PR so its still easy to review this one. Once the other PR is merged, i will merge it into this one and retest it